tests/podman/Dockerfile

```diff
@@ -3,18 +3,17 @@ FROM fedora:latest
 ENV GOPATH=/root/go
 ENV PATH=/usr/bin:/usr/sbin:/root/go/bin:/usr/local/bin::/usr/local/sbin
 
-RUN yum install -y golang python git gcc automake autoconf libcap-devel \
+RUN dnf install -y golang python git gcc automake autoconf libcap-devel \
     systemd-devel yajl-devel libseccomp-devel go-md2man \
-    glibc-static python3-libmount libtool make podman xz nmap-ncat && \
-    dnf install -y 'dnf-command(builddep)' && dnf builddep -y podman && \
-    go get github.com/onsi/ginkgo/ginkgo && \
-    go get github.com/onsi/gomega/... && \
-    mkdir -p /root/go/src/github.com/containers && \
-    chmod 755 /root && \
-    (cd /root/go/src/github.com/containers && git clone -b main https://github.com/containers/podman && \
-     cd podman && \
-     make install.catatonit && \
-     make)
+    glibc-static python3-libmount libtool make podman xz nmap-ncat \
+    'dnf-command(builddep)' && \
+    dnf builddep -y podman && \
+    git clone -b main https://github.com/containers/podman /root/go/src/github.com/containers/podman && \
+    cd /root/go/src/github.com/containers/podman && \
+    go install github.com/onsi/ginkgo/ginkgo && \
+    go install github.com/onsi/gomega/... && \
```

Comment:
Why does these `go install` to have to be after `cd`?

Reply:
Outside of the source tree, `go install` from 1.18 wants a version to be supplied, and I'd rather it just use the version from podman's `go.mod` file.

Reply:
In this case it's probably better to do
```
go install ./vendor/github.com/onsi/ginkgo/ginkgo
go install ./vendor/github.com/onsi/gomega/...
```

or even

```
make .install.ginkgo
```

(apparently installing gomega is not needed since PR #2259).